wallet/src/lmdb_wallet.rs

```diff
@@ -66,7 +67,7 @@ pub struct LMDBBackend<C, K> {
 	db: store::Store,
 	config: WalletConfig,
 	/// passphrase: TODO better ways of dealing with this other than storing
-	passphrase: String,
+	passphrase: ZeroingString,
 	/// Keychain
```

Comment:
Should we change from the 1st door instead of here?  such as in the `inst_wallet()`:
```
	let passphrase = prompt_password(&g_args.password);
```
So, in the `prompt_password()`, `prompt_password_confirm()`, and so on.